proto/dymensionxyz/dymension/lightclient/tx.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package dymensionxyz.dymension.lightclient;
+
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/lightclient/types";
+
+service Msg {
```

Comment:
maybe add a comment say this proto file was cloned to somewhere else so we won't forget